### PR TITLE
Added responsive layout

### PR DESCRIPTION
**Pull request by Oksana-Z**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,88 @@
+<!DOCTYPE html>
+<html lang="en">
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>Responsive Layout</title>
+  <link rel="stylesheet" href="styles/index.css">
+
```
Comment (karmeljuk) [inline index.html:8]: where is a body element?

File: styles/index.css [hunk 2]
```diff
@@ -0,0 +1,189 @@
...
+    grid-template-columns: 1fr 7fr;
+  }
+}
+.page-header div {
+  padding: 0.5em;
+}
+.page-header div:first-child {
+  background-color: #216583;
+}
+.page-header div:last-child {
```
Comment (karmeljuk) [inline styles/index.css]: here and below: use **.class** as a selector

**Review by karmeljuk — CHANGES_REQUESTED** [review]
great code. Minor changes needed

File: index.html (hunk 1 above)
Comment (Oksana-Z) [reply index.html:8]: Google HTML/CSS Style Guide recommends to ommit optional tags. (3.1.7 Optional Tags).
That's why my code without body and head elements.

File: styles/index.css (hunk 2 above)
Comment (Oksana-Z) [reply styles/index.css]: Done.
Thanks for the review.

File: index.html (hunk 1 above)
Comment (karmeljuk) [reply index.html:8]: ok

**Review by karmeljuk — APPROVED** [review]
code ok

**Pull request merged**